constance/apps.py

```diff
@@ -1,11 +1,14 @@
 from django.apps import AppConfig
+from django.core import checks
 from django.utils.translation import gettext_lazy as _
 
+from constance.checks import check_fieldsets
+
 
 class ConstanceConfig(AppConfig):
     name = 'constance'
     verbose_name = _('Constance')
     default_auto_field = 'django.db.models.AutoField'
 
     def ready(self):
-        from . import checks
+        checks.register(check_fieldsets, 'constance')
```

Comment:
There is no need to register check here. They are registered automatically in decorator while loading module. Only import is required here.

Reply:
I just reused practice from django. 

https://github.com/django/django/blob/f302343380c77e1eb5dab3b64dd70895a95926ca/django/contrib/sites/apps.py#L17

I'm not too fond of import which is not clearly understandable.

```
import this
The Zen of Python, by Tim Peters
...
Explicit is better than implicit.
```